pkg/cmd/pkg/package_describe.go

```diff
@@ -14,7 +21,66 @@ func newCmdPackageDescribe(ctx api.Context) *cobra.Command {
 		Short: "Get specific details for packages",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return invokePythonCLI(ctx)
+			packageName := args[0]
+
+			c, err := cosmos.NewClient(ctx, pluginutil.HTTPClient(""))
+			if err != nil {
+				return err
+			}
+			enc := json.NewEncoder(ctx.Out())
+			enc.SetIndent("", "  ")
+
+			if allVersions {
+				versions, err := c.PackageListVersions(packageName)
+				if err != nil {
+					return err
+				}
+				return enc.Encode(versions)
+
+			}
+
+			description, err := c.PackageDescribe(packageName, version)
+			if err != nil {
```

Comment:
I'm not sure if this is the case with describe but from the other cosmos commands, I've found that cosmos will return the exact text that the Python API produced but the errors that the API wrapper returns don't show that when you call `Error()`. For my other PR, I made an error unwrap function to extract these messages (https://github.com/dcos/dcos-core-cli/blob/2.0-patch.x/pkg/cosmos/client.go#L213-L225) which could be useful here.

Reply:
Thanks, I used it